registration.md

```diff
@@ -253,7 +253,14 @@ This describes how we handle the response, after an account has been
 successfully created.
 
 * If the request is `Accept: application/json`, the response should be status
-  200 with an empty body.
+  200 with a JSON body.  The body should only contain a single JSON field:
+  `status`.  It should look something like this:
+
+```json
+{
+  "status": "VERIFIED"
```

Comment:
I'd love to do this -- the initial thing ya'll decided was to NOT return any info though. Do any of you remember why? I'm 100% happy returning all of it, personally.

Tom is on vacation, we can hit him up when he gets back, but if you are ok with this Les I think we should move forward with ALL dataz.